### PR TITLE
Add support to container registry for deleting tags, manifests and repositories

### PR DESCRIPTION
**Pull request by stuartleeks**

### CODE REVIEW

**Comment by lawrencegripper** [comment]
Gonna have a look over this one once I'm back from holidays, if that's alright? Or @jjcollinge can review and merge while I'm away

**Comment by jjcollinge** [comment]
I'll take a look after lunch

**Comment by stuartleeks** [comment]
@jjcollinge - shout if you have any questions

File: internal/pkg/expanders/containerRegistry.go [hunk 1]
```diff
@@ -440,7 +506,7 @@ func (e *ContainerRegistryExpander) getItemsForURL(ctx context.Context, url stri
 	span, ctx := tracing.StartSpanFromContext(ctx, "getItemsForURL(containerregistry):"+url, tracing.SetTag("url", url))
 	defer span.Finish()
 
-	responseBuf, err := e.doRequest(ctx, url, accessToken)
+	responseBuf, err := e.doRequest(ctx, "GET", url, accessToken)
```
Comment (jjcollinge) [inline internal/pkg/expanders/containerRegistry.go]: nit: move these verbs to consts

File: internal/pkg/views/notifications.go [hunk 2]
```diff
@@ -112,11 +112,14 @@ func (w *NotificationWidget) ConfirmDelete() {
...
 
-		swaggerExpander := expanders.GetSwaggerResourceExpander()
-
 		for _, i := range pending {
-			swaggerDeleted, err := swaggerExpander.Delete(context.Background(), i)
-			if err == nil && !swaggerDeleted {
+			var err error
+			fallback := true
+			if i.Expander != nil {
+				deleted, err := i.Expander.Delete(context.Background(), i)
```
Comment (jjcollinge) [inline internal/pkg/views/notifications.go]: general comment: should we be handling contexts a bit better here? Using timeouts and allowing cancellations - something for another PR.

Comment (stuartleeks) [reply internal/pkg/views/notifications.go]: Yeah, that's probably worth raising an issue for!

File: internal/pkg/expanders/containerRegistry.go (hunk 1 above)
Comment (stuartleeks) [reply internal/pkg/expanders/containerRegistry.go]: I'm not sure how much it adds moving these to consts - I find them quite easy to parse (and locate) as strings. I also had a quick look at the net/http code and that just uses plain strings :-)

**Comment by stuartleeks** [comment]
@jjcollinge are you ok to approve?

**Pull request merged**